x-pack/metricbeat/metricbeat.reference.yml

```diff
@@ -1325,6 +1325,9 @@ metricbeat.modules:
   #ssl.certificate_authorities:
   #  - /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
 
+  # Count number of metrics present in Elasticsearch document (default: false)
+  #metrics_count: false
```

Comment:
We have already specified that the default value is `false`. Should we make `true` instead? Like we do for all other fields with default values?

Reply:
No strong opinion on this as this is a reference file and also a metricbeat config. If we are doing it for other fields then it makes sense.

I'll make the change.